tfe/testing.go

```diff
@@ -44,3 +44,13 @@ func skipIfEnterprise(t *testing.T) {
 		t.Skip("Skipping test for a feature unavailable in Terraform Enterprise. Set 'ENABLE_TFE=0' to run.")
 	}
 }
+
+func isAcceptanceTest() bool {
+	return os.Getenv("TF_ACC") == "1"
+}
+
+func skipIfUnitTest(t *testing.T) {
```

Comment:
You may want to add a comment here about how this isn't really necessary for _most_ tests, only ones that don't use the SDK `resource.Test` helper